src/locales/ko/mystery-encounters/clowning-around-dialogue.json

```
@@ -1,34 +1,34 @@
+  "speaker": "어릿광대",
+  "intro_dialogue": "거기있는 어리버리한 바보씨, 화려한 배틀을 맞이할 각오를 하시길!\n이 싸움꾼 광대가 당신을 쓰러트리겠습니다!",
+  "title": "어릿광대의 장난",
+  "description": "이번 상대는 뭔가 이상합니다.\n어릿광대가 당신을 전투에 끌어들이고 싶어하는 것 같은데, 무슨 목적일까요?\n\n{{blacephalonName}}[[은]] 더욱 수상합니다.\n마치 @[TOOLTIP_TITLE]{이상한 타입과 능력을 가진 것 같습니다.}",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Battle the Clown",
-      "tooltip": "(-) Strange Battle\n(?) Affects Pokémon Abilities",
-      "selected": "Your pitiful Pokémon are poised for a pathetic performance!",
-      "apply_ability_dialogue": "A sensational showcase!\nYour savvy suits a sensational skill as spoils!",
-      "apply_ability_message": "The clown is offering to permanently Skill Swap one of your Pokémon's ability to {{ability}}!",
-      "ability_prompt": "Would you like to permanently teach a Pokémon the {{ability}} ability?",
-      "ability_gained": "@s{level_up_fanfare}{{chosenPokemon}} gained the {{ability}} ability!"
+      "label": "어릿광대와 싸운다",
+      "tooltip": "(-) 이상한 배틀\n(?) 포켓몬의 특성 조정",
+      "selected": "당신의 불쌍한 포켓몬들이 한심한 공연을 할 준비가 된 것 같군요!",
+      "apply_ability_dialogue": "센세이셔널한 쇼였습니다!\n당신의 멋진 기지에는 이 센세이셔널한 보상이 잘 어울릴 것 같군요!",
+      "apply_ability_message": "어릿광대가 포켓몬 한 마리의 특성을 영구적으로 {{ability}}로 교환해주겠다고 제안했다!",
+      "ability_prompt": "포켓몬의 특성을 영구적으로 {{ability}} 특성으로 바꾸시겠습니까?",
+      "ability_gained": "@s{level_up_fanfare}{{chosenPokemon}}[[는]] {{ability}} 특성을 얻었다!"
     },
     "2": {
-      "label": "Remain Unprovoked",
-      "tooltip": "(-) Upsets the Clown\n(?) Affects Pokémon Items",
-      "selected": "Dismal dodger, you deny a delightful duel?\nFeel my fury!",
-      "selected_2": "The clown's {{blacephalonName}} uses Trick!\nAll of your {{switchPokemon}}'s items were randomly swapped!",
-      "selected_3": "Flustered fool, fall for my flawless deception!"
+      "label": "도발에 응하지 않는다",
+      "tooltip": "(-) 광대가 화를 냄\n(?) 포켓몬의 아이템 조정",
+      "selected": "한심한 겁쟁이로군요? 즐거운 배틀을 거부하다니!\n이러면 한 방 먹이고 싶어지잖습니까!",
+      "selected_2": "광대의 {{blacephalonName}}[[가]] 무언가 사용한 것 같다!\n{{switchPokemon}}의 모든 아이템이 랜덤하게 재조정된다!",
+      "selected_3": "당황한 바보같은 표정! 완벽한 저의 한 수를 즐겨주시길!"
     },
     "3": {
-      "label": "Return the Insults",
-      "tooltip": "(-) Upsets the Clown\n(?) Affects Pokémon Types",
-      "selected": "Dismal dodger, you deny a delightful duel?\nFeel my fury!",
-      "selected_2": "The clown's {{blacephalonName}} uses a strange move!\nAll of your team's types were randomly swapped!",
-      "selected_3": "Flustered fool, fall for my flawless deception!"
+      "label": "조롱을 되돌려준다",
+      "tooltip": "(-) 광대가 화를 냄\n(?) 포켓몬의 타입 조정",
+      "selected": "한심한 겁쟁이로군요? 즐거운 배틀을 거부하다니!\n이러면 한 방 먹이고 싶어지잖습니까!",
+      "selected_2": "광대의 {{blacephalonName}}[[가]] 무언가 사용한 것 같다!\n멤버 포켓몬들의 타입이 랜덤하게 재조정된다!",
+      "selected_3": "완벽한 저의 속임수! 완벽한 저의 한 수를 즐겨주시길!"
     }
   },
-  "outro": "The clown and his cohorts\ndisappear in a puff of smoke."
+  "outro": "어릿광대는 연기 속으로 사라져 버렸다."
```

```suggestion
  "outro": "어릿광대는 동료 포켓몬과 함께 연기 속으로 사라져 버렸다."
```
아웃트로가
1 선택지 : 동료포켓몬 2마리 기절
2 선택지 : 둘다 생존 -> 스프라이트 3개가 연기속으로 사라짐
3 선택지 : 둘다 생존 -> 스프라이트 3개가 연기속으로 사라짐
이라서
표현 다시 고쳐뒀어요